### ISSUE TITLE
Website shop icon cannot be clicked

### ISSUE THREAD

**Issue by Yehonal** [chromiecraft/chromiecraft#7364]
### Discussed in https://github.com/chromiecraft/chromiecraft/discussions/7363

<div type='discussions-op-text'>

<sup>Originally posted by **TheSCREWEDSoftware** September 17, 2024</sup>
Related to the donations/rewards-catalogue part of the website.

Increase the row by 1 or 2 (1 maybe looks the better) would spread the current 10 elemnts in to a more readable method as screen tend to be wider. 
(Example below)
1 2 3 4 | 1 2 3 4 5
5 6 7 8 | 6 7 8 9 10
9 10 

The interaction of the text should be extended to the icons itself. Still keeping the hover for the value, but allowing the clicking of the images to match the text clicabiltiy.</div>

This is a known website issue and we need to address it to improve our website UX

The priority is to fix the shop icons that cannot be clicked

**Comment by TheSCREWEDSoftware** [comment]
@Yehonal If I've more stuff related to the website, still the shop but which are not bugs but could be improved upon, should i comment here or make a new web issue and call it a suggestion?

**Comment by Helias** [comment]
I think it's better a separate issue with the prefix suggestion, thank you ;-)

**Comment by Yehonal** [comment]
Yes please, different issues. This will be very helpful and we can start
processing them
